application/displayblock.class.inc.php

```diff
@@ -1233,6 +1233,10 @@ protected function RenderActions(array $aExtraParams): iUIBlock
 		} else {
 			$oBlock = DashletFactory::MakeForDashletBadge($sClassIconUrl, $sHyperlink, $iCount, $sClassLabel, null, null, $aRefreshParams);
 		}
+		$sClassDescription = MetaModel::GetClassDescription($sClass);
+		if ($sClassDescription != '') {
```

Comment:
```suggestion
		if (utils::IsNotNullOrEmptyString($sClassDescription)) {
```

I didn't check if it's necessary to add an `@use utils;` statement.